addons/account_intrastat/data/transaction_data.xml

```diff
@@ -1,39 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
   <data noupdate="1">
-    <record id="l10n_be_intrastat.intrastat_transaction_1" model="l10n_be_intrastat.transaction">
+    <record id="account_intrastat_transaction_1" model="account.intrastat.transaction">
       <field name="code">1</field>
```

Comment:
are you sure it's not only valid in Belgium?

Reply:
Same as transport codes, I found exactly the same codes in another countries